rpmio/rpmpgp.h

```diff
@@ -168,7 +169,8 @@ typedef enum pgpPubkeyAlgo_e {
     PGPPUBKEYALGO_EC		= 18,	/*!< Elliptic Curve */
     PGPPUBKEYALGO_ECDSA		= 19,	/*!< ECDSA */
     PGPPUBKEYALGO_ELGAMAL	= 20,	/*!< Elgamal */
-    PGPPUBKEYALGO_DH		= 21	/*!< Diffie-Hellman (X9.42) */
+    PGPPUBKEYALGO_DH		= 21,	/*!< Diffie-Hellman (X9.42) */
+    PGPPUBKEYALGO_EDDSA		= 22	/*!< EdDSA */
```

Comment:
Hmm, this ID isn't officially approved AFAICT, I was only able to find this draft: https://datatracker.ietf.org/doc/draft-ietf-openpgp-rfc4880bis/?include_text=1

If you have another source of information that actually confirms the ID reservation, please add a pointer (to the commit message). Or does being in a draft somehow protect the reservation (I'm not that familiar with IETF processes)? Not sure we should add this as long as it's not official (although since we're not actually generating such content, maybe its not that big a deal)

Reply:
I don't know why this is not accepted yet. The draft is by Werner Koch, the gpg upstream. It's the value gpg uses when you ask it to create a ed25519 pubkey (you need the --expert --full-gen-key options for this).

gpg with EDDSA support is released since some years, so I don't see how the value can change in the future.

Reply:
Ok. Just add a note about this state of affairs (maybe with a reference to the RFC draft) to the commit message, please. Other than that, I have no objections to merging this.

Reply:
I added it to the header file, I hope that's also ok

Reply:
It's now also in the commit message ;)

Reply:
Fair enough :)